tools/mergerer/README.md

```diff
@@ -0,0 +1,20 @@
+Usage:
```

Comment:
Let's, instead of copying the whole repository here, copy only the prebuilt JAR and add instructions on where did it come from. 🙂 Similarly to how we use `jarjar` in `tools` (that's also where the JAR should be placed.